### PR TITLE
Adding comments to deploy file

### PR DESCRIPTION
**Pull request by Goenae**
Adding comments to deploy file

### CODE REVIEW

File: deploy-commands.js [hunk 1]
```diff
@@ -6,10 +6,10 @@ const { clientId, token } = require('./config.json');
 
 // Remember to run node deploy-commands.js to register your commands!
 const commands = [];
-
+// Get all the JS code from commands repository
 const commandsPath = path.join(__dirname, 'commands');
 const commandFiles = fs.readdirSync(commandsPath).filter(file => file.endsWith('.js'));
```
Comment (skyyriiz) [inline deploy-commands.js:11]: Add a comment here

Comment (skyyriiz) [inline deploy-commands.js:11]: Add a comment here

Comment (Goenae) [reply deploy-commands.js:12]: Done

**Comment by skyyriiz** [comment]
nice work tibo

**Pull request merged**